.github/workflows/npm-publish.yml

```diff
@@ -0,0 +1,34 @@
+name: Bahmni Form Controls Package
+
+on:
+  push:
+    branches:
+      - master
+      - 'release-*'
+
+  workflow_dispatch:
+
+jobs:
+  build:
+    runs-on: macos-latest
+    steps:
+      - name: Checkout the code
+        uses: actions/checkout@v3
+      - name: Set up Node JS 10.19.0
+        uses: actions/setup-node@v3
+        with:
+          node-version: 10.19.0
+          registry-url: 'https://registry.npmjs.org'
+      - run: yarn cache clean
+      - name: Install dependencies
+        run: brew install automake autoconf libtool dpkg pkgconfig nasm libpng
+      - name: yarn Install
+        run: yarn
+      - name: yarn test
+        run: yarn test
+      - run: yarn ci
+      - name: Publish the package
```

Comment:
Actually this step should only run when it is a tag push. But okay for now. Anyways once a version is pushed NPM will not allow you to override. @sivareddyp Can you please create a JIRA card to handle this later?

Reply:
Created a Jira card to handle this - https://bahmni.atlassian.net/browse/BS-130